cmd/filegen/main.go

```diff
@@ -24,7 +24,6 @@ import (
 	"github.com/urfave/cli"
 )
 
-const defaultRoundDuration = 5000
```

Comment:
keep const and use it as `roundDuration` default value?

Reply:
would keep it as a single value, because it is not used in any places